AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/Events/ModelSyncedEvent.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+
+/// Used as HubPayload for the `ModelSynced`
+/// - `modelName` (String): the name of the model that was synced
+/// - `isFullSync` (Bool): `true` if the model was synced with a "full" query to retrieve all models
+/// - `isDeltaSync` (Bool): `true` if the model was synced with a "delta" query to retrieve changes since the last sync
+/// - `createCount` (Int): the number of new model instances added to the local store
+/// - `updateCount` (Int): the number of existing model instances updated in the local store
+/// - `deleteCount` (Int): the number of model instances deleted from the local store
```

Comment:
can you move each one to their respective property?

ie.
```
/// The name of the model that was synced
public let modelName: String

/// `true` if the model was synced with a "full" query to retrieve all models
public let isFullSync: Bool
```

Reply:
updated